tests/test_typical_journey.py

```diff
@@ -40,6 +42,20 @@ async def test_happy(
     drs_id = populated_fixture.drs_id
     example_file = populated_fixture.example_file
     joint_fixture = populated_fixture.joint_fixture
+    user_pubkey = encode_key(generate_key_pair().public)
+
+    # simplify testing by using one longer lived work order token
+    work_order_token, pubkey = get_work_order_token(  # noqa: F405
+        file_id=drs_id,
+        user_pubkey=user_pubkey,
+        valid_seconds=120,
+    )
+
+    # modify default headers and patch signing pubkey
+    # mute mypy false positive, dict[str, str] should be compatible with Mapping[str, str]
+    joint_fixture.rest_client.headers = {"Authorization": f"Bearer {work_order_token}"}  # type: ignore
```

Comment:
```suggestion
    joint_fixture.rest_client.headers = Headers(
        {"Authorization": f"Bearer {work_order_token}"})
```

And `from httpx import Headers` at the top.